Animation.cs

```diff
@@ -65,22 +90,31 @@ private void FromGIF(Image gif)
 
         int len = gif.Frames.Count;
 
+        int estimatedSize = Height * Width * len * 4 / 1024 / 1024;
+        if (estimatedSize >= 50)
+            Logger.Log("Loading animation with estimated size more than 50MiB in raw format.");
+
         for (int i = 0; i < len; i++)
         {
-            // ImageSharp returns not milliseconds, but decisecond
+            // ImageSharp returns not milliseconds, but decisecond as documentation says.
             var delay = gif.Frames[i].Metadata.GetGifMetadata().FrameDelay;
 
             // When delay is equal to 1 or less, browsers use a delay equivalent to 100ms.
             // Example: https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/platform/graphics/deferred_image_decoder.cc;drc=e15f3c2e15ee7570159406526444fa1ffb35150f;l=351
             var currentDelay = delay > 1? delay * 10 : 100;
             var currentFrame = gif.Frames.CloneFrame(i);
 
-            Frames.Add(new AnimationFrame { Texture = AssetLoader.TextureFromImage(currentFrame), Delay = TimeSpan.FromMilliseconds(currentDelay) });
+            // ImageSharp image class is not fast enough to draw images in the expected time.
```

Comment:
What is this comment about? How is it related to the code?

Reply:
> What is this comment about? How is it related to the code?

If code were to use only ImageSharp's implementation of the GIF format, background animation play would lag due to the extracting frame delay after the previous frame was played from ImageSharp's object. To reduce play lag I've chosen to read delays on initialization, but not frames, and fill `Frames` collection.

Reply:
Check new comment text, i think it better than previous 1eb0704e8cc0816f4c3285c934c429fb4e2cedb3.